scripts/test_analyzer/compare_coderabbit_decisions.py

```
@@ -0,0 +1,685 @@
+        if result.analyzer.affected_test_count > 10:
+            lines.append(f"- _(and {result.analyzer.affected_test_count - 10} more tests)_")
+        lines.append("")
+
+    # Show analyzer reasoning
+    lines.append(f"**Analyzer reasoning:** {result.analyzer.reason}")
+    lines.append("")
+
+    return lines
+
+
+def generate_markdown_report(results: list[ComparisonResult], repo: str, detailed: bool = False) -> str:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)